cpptod.dd

```diff
@@ -829,6 +831,106 @@ void test()
 }
 ------
 
+<hr>$(COMMENT  -------------------------------------------- )
+
+$(H3 <a name="functioninputparameters">Function Input Parameters</a>)
```

Comment:
Why not `$(LNAME2 functioninputparameters, Function Input Parameters)` (the DDoc macros are a hard to getting used too).